onboard/src/pages/subsections/backgroundInformation/Personalnformation.jsx

```diff
@@ -1,63 +1,112 @@
-            <TextField
-              id="outlined-basic"
-              label="Permanent Address"
-              variant="outlined"
-            />
-            <TextField
-              type="number"
-              id="outlined-basic"
-              label="Primary Phone Number"
-              variant="outlined"
-            />
-            <TextField
-              type="number"
-              id="outlined-basic"
-              label="Secondary Phone Number"
-              variant="outlined"
-            />
-          </>
-    );
+function PersonalInformation() {
+  return (
+    <>
+      <Box sx={{ width: "100%" }}>
+        <Grid container rowSpacing={1} columnSpacing={{ xs: 1, sm: 2, md: 3 }}>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                id="outlined-basic"
+                label="First Name"
+                variant="outlined"
+              />
+            </Item>
+          </Grid>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                id="outlined-basic"
+                label="Last Name"
+                variant="outlined"
+              />
+            </Item>
+          </Grid>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                id="outlined-basic"
+                label="Middle Initial"
+                variant="outlined"
+              />
+            </Item>
+          </Grid>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                type="number"
+                id="outlined-basic"
+                label="SSN"
+                variant="outlined"
+              />
+            </Item>
+          </Grid>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                type="date"
+                id="outlined-basic"
+                label="Birthdate"
+                variant="outlined"
+              />
+            </Item>
+          </Grid>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                id="outlined-basic"
+                label="Current Address"
+                variant="outlined"
+              />
+            </Item>
+          </Grid>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                id="outlined-basic"
+                label="Permanent Address"
+                variant="outlined"
+              />
+            </Item>
+          </Grid>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                type="number"
+                id="outlined-basic"
+                label="Primary Phone Number"
+                variant="outlined"
+              />
+            </Item>
+          </Grid>
+          <Grid item xs={4}>
+            <Item>
+              <TextField
+                type="number"
+                id="outlined-basic"
+                label="Secondary Phone Number"
+                variant="outlined"
```

Comment:
And this one needs phone checking too. Secondary fields may not be required, primary should be.